### PR TITLE
Resolve merge conflicts of #450

### PR DESCRIPTION
**Pull request by mr-kojo**
Resolve merge conflicts of #450.

I don't have permission to push to master, hence a separate pull request...

### CODE REVIEW

File: foodsaving/pickups/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+from foodsaving.pickups.factories import PickupDateFactory
+from foodsaving.pickups.models import PickupDateSeries, PickupDate, Feedback
+from foodsaving.users.factories import UserFactory
+from foodsaving.stores.factories import StoreFactory
+
+
+class TestFeedbackModel(TestCase):
+    @classmethod
+    def setUpClass(cls):
```
Comment (tiltec) [inline foodsaving/pickups/tests/test_models.py]: Ah, you introduced the old style again that Nick removed with #441 - could you fix this? See #425 for more details.

**Comment by tiltec** [comment]
Better not push to master ;) You usually don't need this.
To keep your old PR, just push to the branch as usual. If there's a conflict, you probably did a rebase. Then it's perfectly fine to force-push (`push -f`) to the branch to overwrite it.

File: foodsaving/pickups/tests/test_models.py (hunk 1 above)
Comment (mr-kojo) [reply foodsaving/pickups/tests/test_models.py]: Sorry, this was the result of git's auto-merge, that's why I missed it... I randomly checked some of the auto-merged files but unfortunately not this one.

It's fixed now.

**Comment by tiltec** [comment]
Merged! Let's see how it runs!

**Pull request merged**